streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationIntegrationTest.java

```diff
@@ -645,6 +648,102 @@ public void shouldReduceSessionWindows() throws Exception {
 
     }
 
+    @Test
+    public void shouldCountUnlimitedWindows() throws Exception {
```

Comment:
I've added this semi-randomly because I noticed we had 0 unlimited windows in integration tests at all. If you can think of some place you'd rather test it, I'm happy to redo it.